.github/workflows/ci.yml

```diff
@@ -0,0 +1,89 @@
+name: CI
+
+on:
+  push:
+    branches: [ jgrpp ]
+  pull_request:
+    branches: [ jgrpp ]
+
+jobs:
+  build_linux:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: prepare workspace
+      run: mkdir build
+    - name: install dependencies
+      run: "sudo apt-get install -y zlib1g-dev liblzo2-dev liblzma-dev libpng-dev libfreetype6-dev libfontconfig1-dev libicu-dev libsdl2-dev"
+    - name: cmake
+      run: |
+        pushd build
+        cmake ..
+        popd
+    - name: make package
+      run: |
+        pushd build
+        make package
+        popd
+    - name: upload artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: linux
+        path: build/bundles/*
+
+  build_macos:
```

Comment:
Was testing this build, did not work for me :(

Reply:
it's possible you will need to de-quarantine it yeah. the "simplest" way i found in the past was to `xattr -r -d com.apple.quarantine ~/Downloads/bundle.app`, but iirc you can also command click it to get the gatekeeper prompt. it's also possible you'll need to change the security setting to allow unsigned apps or downloaded apps to run

Reply:
not much we can do about it from this end without a signing key

Reply:
it's also possible of course that this is not the problem at all. i can test this on a vm later